doc/conf.py

```diff
@@ -1,95 +1,87 @@
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# reference documentation build configuration file, created by
-# sphinx-quickstart on Wed Nov 17 12:52:23 2010.
+# Orange3-Network documentation build configuration file, created by
+# sphinx-quickstart on Thu Jul 28 14:12:24 2016.
 #
-# This file is execfile()d with the current directory set to its containing dir.
+# This file is execfile()d with the current directory set to its
+# containing dir.
 #
 # Note that not all possible configuration values are present in this
 # autogenerated file.
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
-import numpydoc
-import imp
-
-#MOCK_MODULES = ['np', 'np.core', 'matplotlib', 'np.core.ma', 'scipy', 'scipy.sparse', 'scipy.optimize', 'scipy.stats', 'ma', 'np.ma', 'linalg', 'np.linalg', 'scipy.linalg']
-#for mod_name in MOCK_MODULES:
-#    sys.modules[mod_name] = mock.Mock()
-
-module_setup = imp.load_source('module_setup', os.path.join(os.path.dirname(__file__), '..', '..', 'setup.py'))
-VERSION = module_setup.VERSION
-AUTHOR = module_setup.AUTHOR
-DOCUMENTATION_NAME = module_setup.DOCUMENTATION_NAME
-
-TITLE = "%s Documentation v%s" % (DOCUMENTATION_NAME, VERSION)
-
+import sys
+import os
+import shlex
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
-import orangecontrib.network
-sys.modules["Orange.network"] = orangecontrib.network
+#sys.path.insert(0, os.path.abspath('.'))
 
-# -- General configuration -----------------------------------------------------
+# -- General configuration ------------------------------------------------
 
-# Add any Sphinx extension module names here, as strings. They can be extensions
-# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.pngmath', 'sphinx.ext.ifconfig', 'numpydoc']
-
-
-# Numpydoc generates autosummary directives for all undocumented members. Orange documentation only includes documented
-# member, so _str_member_list is modified to return [] where a list of undocumented members is originally returned.
-numpydoc.docscrape_sphinx.SphinxDocString._str_member_list # if numpydoc changes, this line will cause an error
-numpydoc.docscrape_sphinx.SphinxDocString._str_member_list = lambda x, y : []
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
 
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx',
```

Comment:
Perhaps add `sphinx.ext.napoleon`. There may be some numpied docstrings. Or they certainly might be in the future.